### PR TITLE
Document Template#figure function

### LINKED ISSUE #647: Document the new Template::figure() method
See https://github.com/contao/contao/pull/1937

### PR DESCRIPTION
**Pull request by m-vo**
closes #647

### CODE REVIEW

File: docs/dev/framework/image-processing/image-studio.md [hunk 1]
```diff
@@ -136,23 +136,69 @@ This can be helpful if you need to output multiple images with a similar configu
...
+There are two ways to output images in your PHP templates:
 
-If you are instead using the legacy Contao `image.html5` template, you can make use of two builtin helper functions to
-apply/get an array of data ready to use. 
+1) **Rendering a `Figure`** &mdash; If you are creating your own PHP templates and want to apply `Figure` data
+   directly, you currently need to create the markup yourself.
 
-```php
-$template = new FrontendTemplate('image');
+   If you are instead using the legacy Contao `image.html5` template, you can make use of two builtin helper functions
```
Comment (fritzmg) [inline docs/dev/framework/image-processing/image-studio.md:144]: Why is the `image.html5` template legacy?

Comment (m-vo) [reply docs/dev/framework/image-processing/image-studio.md:144]: I didn't change that part. :grin:

**Review by fritzmg — APPROVED** [review]

File: docs/dev/framework/image-processing/image-studio.md (hunk 1 above)
Comment (fritzmg) [reply docs/dev/framework/image-processing/image-studio.md:144]: Right :D

**Comment by fritzmg** [comment]
Thank you @m-vo

**Pull request merged**